zerver/webhooks/freshstatus/tests.py

```diff
@@ -0,0 +1,200 @@
+**Scheduled Start Time:** 12/04/2021 17:08 UTC
+**Scheduled End Time:** 12/04/2021 18:08 UTC
+**Affected Services:**
+* Sample Service
+* Sample Service 2
+        """.strip()
+        self.check_webhook(
+            "freshstatus_scheduled_maintenance_planned_multiple_services",
+            expected_topic,
+            expected_message,
+        )
+
+    def test_freshstatus_scheduled_maintenance_planned_multiple_services_over_limit(self) -> None:
+        """
+        Tests if freshstatus scheduled maintenance planned multiple services over limit is handled correctly
+        """
+        expected_topic = "Expect some services downtime due to server maintenance"
+        expected_message = """
+The following scheduled maintenance has been opened: **Expect some services downtime due to server maintenance**
+**Description:** As part of the upgrade routine, we will be carrying out server maintenance work for this Service. This work will affect the Service to be unavailable during the maintenance window. We apologize for any inconvenience this may cause. Please do not hesitate to contact our support team at support@example.com if you have any questions regarding this server upgrading exercise.
+**Scheduled Start Time:** 12/04/2021 17:08 UTC
+**Scheduled End Time:** 12/04/2021 18:08 UTC
+**Affected Services:**
+* Sample Service
+* Sample Service 2
+* Sample Service 3
+* Sample Service 4
+* Sample Service 5
+[and 2 more service(s)]
+        """.strip()
+        self.check_webhook(
+            "freshstatus_scheduled_maintenance_planned_multiple_services_over_limit",
+            expected_topic,
+            expected_message,
+        )
+
+    def test_freshstatus_scheduled_maintenance_note_created(self) -> None:
+        """
+        Tests if freshstatus scheduled maintenance note created is handled correctly
+        """
+        expected_topic = "Scheduled Maintenance Test"
+        expected_message = """
+The following note has been added to the scheduled maintenance: **Scheduled Maintenance Test**
+**Note:** We are about to start the maintenance.
+        """.strip()
+        self.check_webhook(
+            "freshstatus_scheduled_maintenance_note_created", expected_topic, expected_message
+        )
+
+    def test_freshstatus_scheduled_maintenance_closed(self) -> None:
+        """
+        Tests if freshstatus scheduled maintenance closed is handled correctly
+        """
+        expected_topic = "Scheduled Maintenance Test"
+        expected_message = """
+The following scheduled maintenance has been closed: **Scheduled Maintenance Test**
+**Note:** The maintenance is now complete.
```

Comment:
Do we need to repeat the topic here, or could it be just "Scheduled maintenance closed with note:\n{note}"?

Reply:
I think its important to include it like so as that's the main piece of information that the people want to see. The notes are just the addon. And especially as people can force the topic to be static. Its important to include to say which maintenance has been closed.